lib/scidata/caltech101.ex

```diff
@@ -128,6 +129,15 @@ defmodule Scidata.Caltech101 do
       |> Nx.new_axis(-1)
       |> Nx.equal(Nx.tensor(Enum.to_list(1..102)))
 
+  ## Options.
+
+    * `:base_url` - optional. Dataset base URL.
```

Comment:
```suggestion
    * `:base_url` - Dataset base URL.
```
Can we remove "optional" in these function docs across all the modules? I think the rest of the documentation you added makes it clear they're not required :)

Reply:
Sure, let me update this in a minute ;)